File: Sem4/MPP/Lab/ProiectJava/ProjectNetworking/src/main/java/project/network/dto/ExcursieDto.java
package project.network.dto;

import java.io.Serializable;
import java.time.LocalDateTime;

public class ExcursieDto implements Serializable {
	private String obiectivVizitat;
	private String oraPlecare;
	private String firmaTransport;
	private Double pret;
	private int locuriDisponibile;
	private int id;

	public ExcursieDto(String obiectivVizitat, LocalDateTime oraPlecare, String firmaTransport, Double pret, int locuriDisponibile, int id) {
		this.obiectivVizitat = obiectivVizitat;
		this.oraPlecare = oraPlecare.toString();
		this.firmaTransport = firmaTransport;
		this.pret = pret;
		this.locuriDisponibile = locuriDisponibile;
		this.id = id;
	}

	@Override
	public String toString() {
		return "ExcursieDto["  + obiectivVizitat +
				 oraPlecare +
				 firmaTransport +
				 pret +
				 locuriDisponibile +
				 id +
				']';
	}

	public String getObiectivVizitat() {
		return obiectivVizitat;
	}

	public void setObiectivVizitat(String obiectivVizitat) {
		this.obiectivVizitat = obiectivVizitat;
	}

	public LocalDateTime getOraPlecare() {
		return LocalDateTime.parse(oraPlecare);
	}

	public void setOraPlecare(LocalDateTime oraPlecare) {
		this.oraPlecare = oraPlecare.toString();
	}

	public String getFirmaTransport() {
		return firmaTransport;
	}

	public void setFirmaTransport(String firmaTransport) {
		this.firmaTransport = firmaTransport;
	}

	public Double getPret() {
		return pret;
	}

	public void setPret(Double pret) {
		this.pret = pret;
	}

	public int getLocuriDisponibile() {
		return locuriDisponibile;
	}

	public void setLocuriDisponibile(int locuriDisponibile) {
		this.locuriDisponibile = locuriDisponibile;
	}

	public int getId() {
		return id;
	}

	public void setId(int id) {
		this.id = id;
	}
}

File: Sem4/MPP/Lab/ProiectCSharp/networking/ExcursieDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using model;

namespace networking
{
    [Serializable]
    public class ExcursieDTO
    {
        public String obiectivVizitat { get; set; }
        public String oraPlecare {get; set; }
        public String firmaTransport { get; set; }
        public Double pret { get; set; }
        public int locuriDisponibile { get; set; }
        public int id { get; set; }

        public ExcursieDTO(Excursie excursie)
        {
            this.id = excursie.id;
            this.obiectivVizitat = excursie.obiectivVizitat;
            this.oraPlecare = excursie.oraPlecare.ToString();
            this.firmaTransport = excursie.firmaTransport;
            this.pret = excursie.pret;
            this.locuriDisponibile = excursie.locuriDisponibile;
        }
        public ExcursieDTO(string obiectivVizitat, DateTime oraPlecare, string firmaTransport, double pret, int locuriDisponibile, int id)
        {
            this.obiectivVizitat = obiectivVizitat;
            this.oraPlecare = oraPlecare.ToString();
            this.firmaTransport = firmaTransport;
            this.pret = pret;
            this.locuriDisponibile = locuriDisponibile;
            this.id = id;
        }
    }
}
